install/setup/setup-wizard.sh

```diff
@@ -50,14 +50,19 @@ if [ "$install_db" == "y"  ] || [ "$install_db" == "Y" ] ; then
 
     # Only supporting MySql at this time
     database_type=0
-    while [[ ! ${database_type} =~ ^[1-2]{1}$ ]]; do
+    while [[ ! ${database_type} =~ ^[1-3]{1}$ ]]; do
         echo "Which database (Enter the number)?"
         echo "1) MySQL"
         echo "2) PostgresSQL"
+        echo "3) SQLServer"
```

Comment:
Is this option specific to Azure SQL Database or will it also work with a standard Windows SQL Server install? I just want to confirm the name we're using. Also, I think Microsoft always puts a space in the name: SQL Server.